cli/common.h

```diff
@@ -62,10 +62,12 @@ enum switchtec_fw_type check_and_print_fw_image(int img_fd,
 			" * an index (0, 1, 2)\n" \
 			" * a PCI address (3:00.1)\n" \
 			" * an I2C path with slave address (/dev/i2c-1@0x20)\n" \
-			" * a UART path (/dev/ttyUSB0)\n" \
+			UART \
 	}
```

Comment:
I'd probably label the argument EXTRA_HELP, or something to that effect. It could be used for other things.

Reply:
Updated.

Reply:
@lsgunth could you review and approve if it looks OK...